GoogleForCreators/web-stories-wp
# Small UI refinements

## Summary

* ~Hide elements tab behind flag~ -> now done in #2658 / #2616
* Some small UI refinements as per Sam's feedback.
* Disable resizing and collapsing presets panel when there are no presets
* Move alignment options above color presets
* Remove whitepace between drag handle and color swatches
* Expand presets panel when entering edit mode
* Improve grid layout of color swatches 

## Relevant Technical Choices



## To-do



## User-facing changes

<img width="1680" alt="Screenshot 2020-06-19 at 17 19 52" src="https://user-images.githubusercontent.com/841956/85149192-4b153e80-b251-11ea-9b2b-74a8580e49df.png">

<img width="317" alt="Screenshot 2020-06-22 at 13 52 11" src="https://user-images.githubusercontent.com/841956/85284507-9ddc3980-b48f-11ea-8756-a40332186f8f.png">

<img width="302" alt="Screenshot 2020-06-23 at 21 17 31" src="https://user-images.githubusercontent.com/841956/85448533-15e15700-b597-11ea-8d54-cedaca79656b.png">

## Testing Instructions

* ~Verify that elements tab is no longer visible.~ -> now done in #2658 / #2616
* Verify that alignment options are above color presets
* Verify that there is no whitepace between thee presets color swatches and the bottom border
* Verify that a collapsed presets panel is expanded when entering edit mode

---



Fixes #2619
Fixes #2305
See #2616
